### ISSUE TITLE
[nuxt3] Static image generation still results in 404 after latest update for certain images only

### ISSUE THREAD

**Issue by jamesxu123** [nuxt/image#671]
For some reason images in certain routes are not pre-rendered but not others. This is after #614.

I have found that Nuxt Images with `width` and `height` set do not get rendered statically. E.g. `<nuxt-img format="avif" class="img-fluid" width="1920" height="1000" :src="imgPath" alt=""></nuxt-img>` does not, but `<nuxt-img format="avif" class="img-fluid" :src="imgPath" alt=""></nuxt-img>` does.

**Comment by jamesxu123** [comment]
Example: https://stackblitz.com/edit/github-3bv9kb?file=app.vue

**Comment by jamesxu123** [comment]
Static generation does not seem to be working at all when using Nuxt Picture

**Comment by jimhlad** [comment]
@jamesxu123 Does this only happen when multiple formatters are applied (e.g. width, height, quality, etc)?

If so just made #673 to hopefully solve this issue. And #674 possibly for the second one.

**Comment by FilipBartos** [comment]
@jimhlad I am using simple `<nuxt-picture>` in my project:

```vue      
<nuxt-picture
  :src="..."
  :imgAttrs="..."
/>
```

and your changes in #674 seems to fixed image generation for me.  I hope these changes will be merged soon 👍 

Thanks!

**Comment by tex0l** [comment]
On my end, it doesn't work at all, no static image is built. Am I missing something?

My env:
- `node v18.12.1`
- `macOS 13.0.1 (22A400)`
- `nuxt@3.0.0`
- `@nuxt/image-edge@1.0.0-27827328.bc9ddc0`

I use as well (normally unrelated):
- `@nuxt/content@2.2.2`
- `nuxt-icon@0.1.8`
- `@nuxtjs/tailwindcss@6.1.3` along with `@tailwindcss/line-clamp@0.4.2`
- `sass@1.56.1`

My `@nuxt/image-edge` config is as follows:
```
// nuxt.config.ts
  image: {
    dir: 'assets/images'
  }
```

I use `nuxt-picture` or `nuxt-img` like this:
`<nuxt-img src="image-name.png"/>` or `<nuxt-picture src="image-name.png" />`

And they are rendered as:
```html
<img src="/_ipx/_/image-name.png">
```

 and 

```html
<picture>
  <source
    type="image/webp"
    sizes="(max-width: 320px) 320px, (max-width: 640px) 640px, (max-width: 768px) 768px, (max-width: 1024px) 1024px, (max-width: 1280px) 1280px, (max-width: 1536px) 1536px, 1536px"
    srcset="/_ipx/w_320,f_webp/image-name.png 320w, /_ipx/w_640,f_webp/image-name.png 640w, /_ipx/w_768,f_webp/image-name.png 768w, /_ipx/w_1024,f_webp/image-name.png 1024w, /_ipx/w_1280,f_webp/image-name.png 1280w, /_ipx/w_1536,f_webp/image-name.png 1536w, /_ipx/w_1536,f_webp/image-name.png 1536w">
  <img
    src="/_ipx/w_1536,f_png/image-name.png"
    sizes="(max-width: 320px) 320px, (max-width: 640px) 640px, (max-width: 768px) 768px, (max-width: 1024px) 1024px, (max-width: 1280px) 1280px, (max-width: 1536px) 1536px, 1536px"
    srcset="/_ipx/w_320,f_png/image-name.png 320w, /_ipx/w_640,f_png/image-name.png 640w, /_ipx/w_768,f_png/image-name.png 768w, /_ipx/w_1024,f_png/image-name.png 1024w, /_ipx/w_1280,f_png/image-name.png 1280w, /_ipx/w_1536,f_png/image-name.png 1536w, /_ipx/w_1536,f_png/image-name.png 1536w">
</picture>
```

But in my `.output/public` there is no `_ipx/` directory.

By the way, I get the following error during prerender:
```
ERROR  (node:20632) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time 
(Use `node --trace-warnings ...` to show where the warning was created)
```

I don't know if that's expected.

**Comment by papadi166** [comment]
Same problem no `_ipx/ ` directory

**Comment by pi0** [comment]
Hi there! Sorry for checking this too late. Some improvements are landed via #725 to v1 edge which i believe should solve issues related to prerendering. Please try upgrading nuxt+nitro+image and let me know with a new issue/reproduction if still some images are not properly prerendered.
